extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-attachment.js

```diff
@@ -17,27 +17,38 @@ describes.realWin(
 
     beforeEach(async () => {
       win = env.win;
+      registerServiceBuilder(win, 'performance', function () {
+        return {
+          isPerformanceTrackingOn: () => false,
+        };
+      });
+      env.sandbox.stub(win.history, 'replaceState');
       await createAmpStoryShoppingAttachment();
     });
 
     async function createAmpStoryShoppingAttachment() {
+      const story = win.document.createElement('amp-story');
+      win.document.body.appendChild(story);
       const pageEl = win.document.createElement('amp-story-page');
       pageEl.id = 'page1';
-      element = createElementWithAttributes(
-        win.document,
-        'amp-story-shopping-attachment',
-        {'layout': 'nodisplay'}
-      );
+      element = win.document.createElement('amp-story-shopping-attachment');
```

Comment:
Optional: the naming of the `element` and `shoppingAttachment` variables might be a bit clearer to me if they were instead updated to `attachmentEl` and `attachmentImpl`. I don't feel too strongly about it, so it's up to you!

Reply:
Good call. Updated.